web/src/main/webapp/WEB-INF/jsp/managestudy/viewSectionDataPreview.jsp

```diff
@@ -15,318 +15,317 @@
+                                        <div class="tab_BG_h">
+                                            <div class="tab_R_h" style="padding-right: 0px;">
+                                                <div class="tab_L_h" style="padding: 3px 11px 0px 6px; text-align: left;">
+                                                    <a href="javascript:leftnavExpand('CRF_infobox_closed'); leftnavExpand('CRF_infobox_open');"><img src="images/sidebar_collapse.gif" border="0" align="right"></a>
+                                                    <b><fmt:message key="crf_info" bundle="${resword}"/></b>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </td>
+                                </tr>
+                            </table>
+                        </td>
+                    </tr>
+                    <tr>
+
+                        <td valign="top">
+                            <div class="box_T">
+                                <div class="box_L">
+                                    <div class="box_R">
+                                        <div class="box_B">
+                                            <div class="box_TR">
+                                                <div class="box_BL">
+                                                    <div class="box_BR">
+
+                                                        <div class="tablebox_center">
+
+
+                                                            <table border="0" cellpadding="0" cellspacing="0" width="650" style="border-style: solid; border-width: 1px; border-color: #CCCCCC;">
+                                                                <tr>
+                                                                        <c:out value="${section.crf.name}"/> <c:out value="${section.crfVersion.name}"/>&nbsp;&nbsp;
+                                                                    <c:choose>
+                                                                    <c:when test="${EventCRFBean.stage.name=='initial data entry'}">
+                                                                    <img src="images/icon_InitialDE.gif" alt="<fmt:message key="initial_data_entry" bundle="${resword}"/>" title="<fmt:message key="initial_data_entry" bundle="${resword}"/>">
+                                                                    </c:when>
+                                                                    <c:when test="${EventCRFBean.stage.name=='initial data entry complete'}">
+                                                                    <img src="images/icon_InitialDEcomplete.gif" alt="<fmt:message key="initial_data_entry_complete" bundle="${resword}"/>" title="<fmt:message key="initial_data_entry_complete" bundle="${resword}"/>">
+                                                                    </c:when>

```

Comment:
This key translates to: "Initial Data Entry Completed". Need to create a new one that translates to "Initial Data Entry Complete"